programs/svm-spoke/src/instructions/deposit.rs

```diff
@@ -255,9 +253,8 @@ pub fn unsafe_deposit_v3(
 pub struct Null {}
 pub fn get_unsafe_deposit_id(msg_sender: Pubkey, depositor: Pubkey, deposit_nonce: u64) -> [u8; 32] {
     let mut data = Vec::new();
-    data.extend_from_slice(&msg_sender.to_bytes());
-    data.extend_from_slice(&depositor.to_bytes());
-    data.extend_from_slice(&deposit_nonce.to_le_bytes());
+    // Use AnchorSerialize to serialize the tuple of values
+    (msg_sender, depositor, deposit_nonce).serialize(&mut data).unwrap();
```

Comment:
note that this is not AnchorSerialize wrapper (like mentioned in comment above), but using borch directly. Though, I guess functionally its the same in this case.

Reply:
done.